test/lib/cli/commands/serve.js

```diff
@@ -343,3 +347,22 @@ test.serial("ui5 serve: --framework-version", async (t) => {
 		}
 	}]);
 });
+
+test.serial("ui5 serve: --serve-csp-reports=true", async (t) => {
```

Comment:
```suggestion
test.serial("ui5 serve --serve-csp-reports", async (t) => {
```

Like in the other tests: No colon and boolean arguments don't need a value

Reply:
done